src/test/scala/nl/knaw/dans/deposit/DepositSpec.scala

```diff
@@ -770,6 +777,20 @@ class DepositSpec extends TestSupportFixture
     resultDeposit.isDoiRegistered shouldBe empty
   }
 
```

Comment:
no tests for the getter of `dansDoiAction`?

Reply:
The getter is tested implicit in the setter and unsetter test. I added an explicit test for it.